doc/supported-platforms.md

```diff
@@ -0,0 +1,16 @@
+# Supported Platforms #
+
+Ignition is currently only supported for the following platforms:
+
+- [Bare Metal](https://coreos.com/os/docs/latest/installing-to-disk.html) - Use
+  the `coreos.config.url` kernel parameter to provide a URL to the
+  configuration. The URL can use the `http://` schema to specify a remote
```

Comment:
it's pretty well established to use `scheme` here rather than `schema`:
https://en.wikipedia.org/wiki/Uniform_resource_locator#Syntax